### PR TITLE
Add app lock notify commands 

### PR DESCRIPTION
**Pull request by RoboMagus**
For PR home-assistant/android#2868

### CODE REVIEW

**Comment by dshokouhi** [comment]
Lets make sure to add the beta flag to the table and the actual description so users know its part of the beta

**Review by dshokouhi — CHANGES_REQUESTED** [review]
Missing the beta flag `<span class='beta'>BETA</span>` in the table and description. As features get merged they are first part of the beta until a release is cut.

File: docs/notifications/commands.md [hunk 1]
```diff
@@ -102,6 +103,36 @@ Please see the below table for new parameters to use after updating the Android
...
 
+## App lock
+
+![Android](/assets/android.svg) <span class='beta'>BETA</span>
+
+To take control of an Android companion app's security Users can alter the app lock settings using `message: command_app_lock`. All settings related to the app lock can be configured in a single command. The following settings are accessible through the notify command:
+
+| Parameter | Type | Description |
+|---------|---------|--------|
+| `app_lock_enabled` | boolean | Wether the biometric / screenlock will be enabled |
```
Comment (dshokouhi) [inline docs/notifications/commands.md:114]:
```suggestion
| `app_lock_enabled` | boolean | Whether the biometric / screen lock will be enabled |
```

File: docs/notifications/commands.md [hunk 2]
```diff
@@ -102,6 +103,36 @@ Please see the below table for new parameters to use after updating the Android
...
+| `app_lock_timeout` | integer | Session timeout in seconds |
+| `home_bypass_enabled` | boolean | Wether the lock is bypassed when connected to home WiFi |
```
Comment (dshokouhi) [inline docs/notifications/commands.md:116]:
```suggestion
| `home_bypass_enabled` | boolean | Whether the lock is bypassed when connected to home WiFi |
```

**Comment by dshokouhi** [comment]
2 small typos, sorry for not catching that earlier :)

**Comment by RoboMagus** [comment]
Np, thanks for being so active in guiding the development!

**Review by dshokouhi — APPROVED** [review]
Looks good to me can be merged when the parent is merged

**Comment by dshokouhi** [comment]
@RoboMagus  just a FYI but there is a conflict in this file now

**Pull request merged**